src/test/stub-tests/HashSet/main.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// rmc-flags: --use-abs --abs-type c-ffi
+include!{"../../rmc-prelude.rs"}
+
+fn main() {
+    let mut h: HashSet<u16> = HashSet::new();
+
+    assert!(h.insert(10));
```

Comment:
Use nondet values here would be neater

Reply:
```rust
let a = __nondet();
let b = __nondet();
__VERIFIER_assume(a != b);
```

Reply:
Make another version of the test, mark skip, file ticket for known performance issue

Reply:
Created issue: https://github.com/model-checking/rmc/issues/478 for tracking